tests/src/python/test_authsettingswidget.py

```diff
@@ -46,7 +46,7 @@ def setUpClass(cls):
         Creates an auth configuration"""
         # Enable auth
         # os.environ['QGIS_AUTH_PASSWORD_FILE'] = QGIS_AUTH_PASSWORD_FILE
-        authm = QgsAuthManager.instance()
+        authm = QgsApplication.authManager()
         assert (authm.setMasterPassword('masterpassword', True))
```

Comment:
@elpaso so how does this not fail if `QgsAuthManager` is not imported, i.e. call to `setMasterPassword`, as well as `storeAuthenticationConfig` etc? Must be pulled in by another `QgsAuth*` module

Reply:
@dakcarto that's how python works, if you access a member variable through its class instance you don't need to import the module where the member variable is defined.